### ISSUE TITLE
Request - remote UI call to open a specific blueprint in manufacturing window

### ISSUE THREAD

**Issue by wtfrank** [esi/esi-issues#59]
When you have multiple BPCs of one type with different runs remaining, ME and TE, it would be useful to be able to remotely open a particular BPC in the industry window (similarly to remotely opening a specific contract). This would make it easier to set off jobs with the blueprint you intended to, instead of building from a too low ME BPC and running out of materials, or leaving an inconvenient number of runs remaining on a BPC.

**Comment by wtfrank** [comment]
This would perhaps have the side-effect of avoiding some of the performance issues that the industry window suffers from once you build up a small collection of blueprints - particularly the search box lag.

**Comment by aquarhead** [comment]
Hmm.. how would you identify the different BPCs? Can you get their item IDs or what?

**Comment by wtfrank** [comment]
Yes you are given their item ids by the blueprints api, and by the assets api (less usefully though as only the blueprints api gives you the ME/TE/runs attributes).

**Comment by aquarhead** [comment]
Cool, noted.

**Comment by a-tal** [comment]
approved by GD

**Comment by SpeedProg** [comment]
This is gonna support opening BPOs too?

**Comment by Meigs2** [comment]
Hello, I'm working on an industry production/planning tool and it'd be really neat to see an endpoint that lets you open the industry window, similar to the market window. I'd like to allow a "checklist"-style job installation feature similar to eve-tycoon opening the market browser + copying price to your clipboard. The user would use a global hotkey to invoke this endpoint when installing jobs, which would move to the "next" job to install.

Allowing the user to specify a blueprint_id returned from `/characters/{character_id}/blueprints` would also give planning tools the ability to fully plan production runs, down to the individual blueprint and their availability/runs.

An implementation of the endpoint might look something like the following:

---

Route: `POST: https://esi.evetech.net/ui/openwindow/industry`

Attempts to open the industry window with a set of given filter(s).

### Basic Parameters:
* `blueprint_id: (optional)`: The the id of the blueprint to be selected. If the blueprint is being used as part of an active job, that job should be shown.
* `job_id (optional)`: The industry job_id to open.

### Advanced Parameters (nice to haves)
* `facility_id (optional)`: Pre-selects the specified industry facility.
* `activity_id (optional)`: Pre-selects the specified industry activity.
* `input_location_id (optional)`: Sets the location_id where materials for the job are sourced from.
* `output_location_id: (optional)`: Sets the output_location_id where the job will output to.
* `runs (optional)`: If specified, attempts to set the number of runs for the job. (this would be amazingly nice to have, but I'd understand why it might not be wanted to be added.)

### Invention Parameters
* `outcome_type_id`: Pre-selects the outcome of invention, if multiple choices are available.
* `decryptor_type_id`: Pre-selects the decryptor to use as part of an invention job, if able.

---

Optionally, you could separate out the installation action(s) to be per-activity, so the number of optional parameters doesn't explode. For example:

`POST: https://esi.evetech.net/ui/openwindow/industry/{blueprint_id}/invention`

Would include the outcome_type_id and decryptor_type_id parameters, but 

`POST: https://esi.evetech.net/ui/openwindow/industry/{blueprint_id}/copying`

Would not.

Thanks!
